dev-python/unasync/unasync-0.5.0.ebuild

```diff
@@ -20,5 +20,5 @@ src_prepare() {
 	default_src_prepare
 }
 
-distutils_enable_sphinx docs/source dev-python/sphinx_rtd_theme
+distutils_enable_sphinx docs/source dev-python/sphinxcontrib-trio dev-python/sphinx_rtd_theme
```

Comment:
Is there any guidance on this in documentation? As the sed line  [here](https://github.com/gentoo/gentoo/pull/22740/files#diff-6f82d55e41e43fd29ac926cc9c6b2477f1113c25cb5767a233209f31baa3af01L19) is 129 characters long, should that be broken up into multiple lines as well?

Reply:
It isn't just because of a long line, but also more cleaner dependency list. The same reason "*DEPEND" variables are broken into one at a line, it is better to break here.

While for one package it is accepted to be written in the same line as `distutils_enable_sphinx`, when two are in the same, line, it is harder to read and list.

---

By the way, for the sed line, After reviewing the test file [here](https://github.com/python-trio/unasync/blob/master/tests/test_unasync.py), I think we can compress the whole sed into:
`sed -e '/PYTHONPATH/d' -i tests/test_unasync.py || die`

Reply:
Ammended commit with these changes. Apologies for letting this sit for such an over extended time.